ui/pages/sample/components/sample-counter-pane.tsx

```diff
@@ -0,0 +1,47 @@
+import React from 'react';
+import { Box, Button, Text } from '../../../components/component-library';
+import Card from '../../../components/ui/card';
+import {
+  Display,
+  FlexDirection,
+  AlignItems,
+  TextAlign,
+  TextVariant,
+  BlockSize,
+} from '../../../helpers/constants/design-system';
+import { useSample } from '../../../ducks/sample';
+
+export function SampleCounterPane() {
+  const counter = useSample();
+
+  return (
+    <Card
+      data-testid="sample-counter-pane-card"
+      width={BlockSize.Full}
+      style={{ maxWidth: '500px' }}
+    >
+      <Box
+        display={Display.Flex}
+        flexDirection={FlexDirection.Column}
+        alignItems={AlignItems.center}
+        gap={4}
+      >
+        <Text
+          variant={TextVariant.headingSm}
+          data-testid="sample-counter-pane-title"
+        >
+          Counter
+        </Text>
+        <Text data-testid="sample-counter-pane-value">{`Value: ${counter.value}`}</Text>
+        <Button
+          onClick={() => counter.increment()}
+          textAlign={TextAlign.Center}
+          data-testid="sample-counter-pane-increment-button"
+          block
+        >
+          Increment Redux Counter
+        </Button>
+      </Box>
+    </Card>
+  );
+}
```

Comment:
I'm of the opinion that all UI files should have an accompanying storybook file large or small this greatly help check for system wide changes